app/services/create_court.rb

```diff
@@ -0,0 +1,56 @@
+class CreateCourt
+  def initialize(params)
+    @params = params
+  end
+
+  def process
+    court = Court.create!(court_attributes.merge(params))
+    Reservation.create!(school_reservation: true, court_id: court.id,
```

Comment:
przerób to na funkcje, w której będziesz podawał time_from i time_to) i wtedy bedziesz tworzył rezerwacje

Reply:
metode, jesli juz

Reply:
Słusznie.

Reply:
No to co Daek proponuje albo jakąś iterację, każde wywołanie się różni tylko dniem tygodnia, da się to ujednolicić na pewno